amis_cf_sm.md

```diff
@@ -2,27 +2,29 @@
 
 This document covers how to find AMIs compatible with AWS Graviton, and how to look up and use the AMIs in AWS System Manager and AWS CloudFormation.
 
-## Amazon Linux (AL2 and AL2023)
+## Using the console
 
-### Using the console
-
-**Amazon Linux 2023** and **Amazon Linux 2** AMIs can both be found in the
-console as explained in the [AWS
-documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/finding-an-ami.html#finding-an-ami-console)
+AMIs can both be found in the console as explained in the [AWS documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/finding-an-ami.html#finding-an-ami-console)
 when launching instances interactively.
 
-### Using APIs - AWS Systems Manager Parameter Store
+## Using APIs - AWS Systems Manager Parameter Store
 
 When integrating the AMI lookup process in a script or in a piece of code, it is more convenient to leverage [AWS Systems Manager](https://aws.amazon.com/systems-manager/) Parameter Store.
 
 There's a good article about it on the AWS Compute blog: [Query for the latest Amazon Linux AMI IDs using AWS Systems Manager Parameter Store](https://aws.amazon.com/blogs/compute/query-for-the-latest-amazon-linux-ami-ids-using-aws-systems-manager-parameter-store/).
 
-
 |OS release|Parameter name|
```

Comment:
It is better to point to a page which has list of OS supported instead of listing it here. Do you have that @arthurpetitpierre ? Thanks

Reply:
I'm not aware of such a page maintained by AWS. There's one for the various versions of Amazon Linux, but none for other OS vendors.
Listing the SSM parameters for other distributions here allows people migrating workloads to Graviton to update the way they pull AMI ids at the same time.